site/en/workflows.md

```diff
@@ -64,7 +64,7 @@ the raw image is taken and enhanced by e.g. grayscale conversion, brightness nor
 	  for parameters and command examples (presets) see [the Readme](https://github.com/bertsky/ocrd_wrap#ocr-d-processor-interface-ocrd-preprocess-image)
 	  </td>
       <td><code>
-	  ocrd-preprocess-image -I OCR-D-IMG -O OCR-D-PREP -p '{"output_feature_added": "binarized","command": "scribo-cli sauvola-ms-split '@INFILE' '@OUTFILE' --enable-negate-output"}'
+	  ocrd-preprocess-image -I OCR-D-IMG -O OCR-D-PREP -P output_feature_added binarized -P command "scribo-cli sauvola-ms-split @INFILE @OUTFILE --enable-negate-output"
```

Comment:
You tested this, right? We don't need quotes around `@INFILE`/@OUTFILE`?

Reply:
yes, it works like this. also works with single quotes though